lib/matplotlib/text.py

```diff
@@ -307,7 +306,10 @@ def set_rotation_mode(self, m):
             aligned according to their horizontal and vertical alignments.  If
             ``"anchor"``, then alignment occurs before rotation.
         """
-        _api.check_in_list(["anchor", "default", None], rotation_mode=m)
+        if m is None:
+            m = "default"
+        else:
+            _api.check_in_list(("anchor", "default"), rotation_mode=m)
```

Comment:
If this change is OK one should probably clarify that setting `None` will return `"default"` in the doc-string.